malcontent.yaml

```diff
@@ -38,6 +34,7 @@ test:
       packages:
         - openssl
         - crane
+        - yara
```

Comment:
Should this be `yara-x`?

Reply:
`yara` is appropriate currently because the test runs `mal analyze /usr/bin/yara`, which was previously implicitly installed as a dependency of the package under test.

Reply:
Ah okay, I misread. Thanks!